pubsub/pubsub.go

```diff
@@ -38,18 +38,25 @@ var (
 	ErrAlreadySubscribed = errors.New("already subscribed")
 )
 
+type ITagSet interface {
+	Get(key string) (result interface{}, found bool)
+	Len() int
+}
+
+type TagSet map[string]interface{}
```

Comment:
do we need to export this?? see my comment above

Reply:
Don't know what it is used for but if you think it should be private so be it.